src/test/java/Vat/VatRepositoryTest.java

```diff
@@ -20,4 +22,18 @@ void shouldThrowExceptionWhenCountryIsIncorrect() {
         //then
         assertThat(exception.getMessage()).isEqualTo("Vat for country Poland111 and product type BOOK was not found");
     }
+    
+	@Test
+	void shouldReturnTheSameSizeOfSetWhenAddExistingObj(){
+		//given
+		VatRepository repositoryToTest = new VatRepository();
+		
+		//when
+		int sizeBefore = repositoryToTest.vatValues.size();
+		repositoryToTest.addVatValue( "Poland", Type.BOOK, new BigDecimal( "0.05" ) );
```

Comment:
I would change this test a bit. Maybe add something new twice. Then, if the rest changes (f.e. like Polish books), the test will still work.